src/test/resources/features/US07_DeleteFolder_TD.feature

```diff
@@ -13,8 +13,8 @@ Feature: User should be able to see all deleted file, delete and restore files
 
   @US07-AC2
   Scenario: user delete permanently any deleted files
-    And user deletes "Talk" file permanently by using ellipses in the file's line
-    Then user do not see deleted "Talk" file
+    And user deletes "klasor" file permanently by using ellipses in the file's line
+    Then user do not see deleted "klasor" file
 
   @US07-AC3
   Scenario: user can restore any files and see it on All files
```

Comment:
Restoring the folder "Dilo" breaks your flow , when I run the test from the first AC , it fails because the folder already exists

Reply:
![Screenshot 2024-11-13 at 12 03 50 AM](https://github.com/user-attachments/assets/9a4884bf-4d47-4a8d-9c9e-0b1d46118142)

Reply:
![image](https://github.com/user-attachments/assets/0df9c75c-6ae1-40b6-8996-7349eab6cec0)

Reply:
I think i have to delete then